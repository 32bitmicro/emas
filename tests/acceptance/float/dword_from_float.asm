.dword 5237846.0, 99999999999999999.0/125768923476
