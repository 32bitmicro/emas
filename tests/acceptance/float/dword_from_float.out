@ 0x0000 : 0x004f  /  000 000 0 001 001 111  /  79
@ 0x0001 : 0xec56  /  111 011 0 001 010 110  /  60502
@ 0x0002 : 0x04e7  /  000 001 0 011 100 111  /  1255
@ 0x0003 : 0xfff5  /  111 111 1 111 110 101  /  65525
